src/sections/Landing/Landing.jsx

```diff
@@ -1,13 +1,15 @@
 import "./Landing.scss";
 import techNovaText from "../../resources/images/techNovaText.svg";
+import CountDown from "./countdown";
 
 const Landing = () => {
   return (
     <div className="banner">
+      <CountDown />
```

Comment:
Could you move the countdown between the Technova words + Women+ in Tech Hackathon line and the mailing list line to match the Figma design that Fariha used?

Reply:
Currently, the white words overlap with the white sign. While you're at it, could you add some space between parts like how it is on Figma?